crates/driver/example.toml

```diff
@@ -34,10 +34,11 @@ use-soft-cancellations = true
 [contracts] # Optionally override the contract addresses, necessary on less popular blockchains
 gp-v2-settlement = "0x9008D19f58AAbD9eD0D60971565AA8510560ab41"
 weth = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
-# TODO replace with the actual address when the contract actually gets replaced
-flashloan-wrappers = ["0x0000000000000000000000000000000000000000"]
 flashloan-router = "0x0000000000000000000000000000000000000000"
 
+[contracts.flashloan-wrapper-by-lender."0x0000000000000000000000000000000000000000"]
+address = "0x0000000000000000000000000000000000000000"
```

Comment:
Oh, my god. This is how `yaml` stores structs in a map?! 😆 

WDYT about passing this as a list? I believe that would look sth like:
```
[[flashloan-adapter]]
lender = "0x0000000000000000000000000000000000000000"
helper-contract = "0x0000000000000000000000000000000000000000"
fee-in-bps = 5
```
It's a bit more verbose but also less weird to look at. 😅